src/histolab/exceptions.py

```diff
@@ -37,6 +37,10 @@ class LevelError(HistolabException):
     """Raised when a requested level is not available"""
 
 
+class PropertyError(HistolabException):
```

Comment:
I would call it with the `Slide` prefix considering that the docstring says :`Raised when a requested slide property is not available` . So if it is a specific Exception for a slide I would use that prefix

`SlidePropertyError`